python/src/pyarrow/helpers.h

```diff
@@ -31,6 +31,9 @@ using arrow::Type;
 PYARROW_EXPORT
 std::shared_ptr<DataType> GetPrimitiveType(Type::type type);
 
+PYARROW_EXPORT
+std::shared_ptr<DataType> GetTimestampType(arrow::TimeUnit unit);
```

Comment:
Yes, there is actually no need for the `helpers.cc/h` except that I failed to include it correctly in `pyarrow/includes/libarrow.pxd`.